pkg/logs/config/integration_config.go

```diff
@@ -70,6 +70,17 @@ type IntegrationConfig struct {
 	Logs []IntegrationConfigLogSource
 }
 
+// GetWildcardOpenFilesLimit returns the maximum number of files that can be opened at the same time
+func GetWildcardOpenFilesLimit() int {
+	return getWildcardOpenFilesLimit(LogsAgent)
+}
+
+// getWildcardOpenFilesLimit returns the value specified in the configuration file,
+// if it does not exist, it returns 0 by default
```

Comment:
we should set a default above zero, but I suspect this is temporary :)

Reply:
There is a default value `defaultTailingLimit` set to `100` in `scanner.go`